### PR TITLE
Fix 404s

### PR DESCRIPTION
**Pull request by ligi**

### CODE REVIEW

**Comment by monkey-mas** [comment]
@oldergod @Egorand 
Is there any specific reason why we have `sed` command in `deploy_website.sh`? if we have the valid URL in `README` like the fix of this PR, I think we no longer need the `sed`.

```bash
  # Fix *.md links to point to where the docs live under Mkdocs.
  # Linux
  # sed -i 's/docs\/wire_compiler.md/wire_compiler/' README.md
  # OSX
  sed -i "" 's/docs\/wire_compiler.md/wire_compiler/' README.md
  sed -i "" 's/docs\/wire_grpc.md/wire_grpc/' README.md
```

File: README.md [hunk 1]
```diff
@@ -603,8 +603,8 @@ See [Google's excellent documentation][schema_docs] on the structure and syntax
  [google_protos]: https://developers.google.com/protocol-buffers/docs/overview
  [effective_java]: https://www.amazon.ca/Effective-Java-3rd-Joshua-Bloch/dp/0134685997/
  [schema_docs]: https://developers.google.com/protocol-buffers/docs/proto
- [compiler_docs]: docs/wire_compiler.md
- [grpc_docs]: docs/wire_grpc.md
+ [compiler_docs]: https://square.github.io/wire/wire_compiler/
+ [grpc_docs]: https://square.github.io/wire/wire_grpc/
```
Comment (oldergod) [inline README.md]:
```suggestion
 [compiler_docs]: wire-library/docs/wire_compiler.md
 [grpc_docs]: wire-library/docs/wire_grpc.md
```
This is probably what we want.

**Comment by oldergod** [comment]
Could you sign [our CLA](https://docs.google.com/forms/d/e/1FAIpQLSeRVQ35-gq2vdSxD1kdh7CJwRdjmUA0EZ9gRXaWYoUeKPZEQQ/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1) so we can merge your PR?

**Comment by oldergod** [comment]
@monkey-mas The _true_ fix points to `README`s in the code so the `sed` is still required.

**Comment by ligi** [comment]
Sorry - I cannot sign this CLA - it wants way to personal data from me

**Comment by oldergod** [comment]
Thanks anyway @ligi I'll fix it then.

**Pull request closed without merging**